dev/test/bundle.ts

```diff
@@ -80,11 +80,14 @@ describe('Bundle Buidler', () => {
     expect(elements.length).to.equal(3);
 
     const meta = (elements[0] as IBundleElement).metadata;
+    expect(meta!.totalBytes).greaterThan(0);
```

Comment:
Is the size not fixed here in the unit tests? Or will that make the tests too fragile?

Reply:
Yes, I started with checking for exact size, but it became too fragile.